src/styles/global.ts

```diff
@@ -0,0 +1,31 @@
+import { createGlobalStyle, css } from 'styled-components'
+
+const GlobalStyles = createGlobalStyle`
+@font-face {
+  font-family: 'Lato';
+  font-style: normal;
+  font-weight: 300;
+  font-display: swap;
+  src: local('Lato Light'), local('Lato-Light'),
+       url('/fonts/lato.light.ttf');
+}
```

Comment:
Since this is a font google, I particularly prefer to use straight from google, it is easier to manage if they have multiple sources in the project, and is delivered by CDN, you will not burden your hosting to deliver this file.

https://fonts.google.com/specimen/Lato?query=Lato